### PR TITLE
Update pricing info for Reports API

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Update pricing information.

JIRA: https://nexmoinc.atlassian.net/browse/DEVX-4304

## Review

* [Review](https://nexmo-developer-pr-3205.herokuapp.com/reports/overview#pricing)

### CODE REVIEW

File: _documentation/en/reports/overview.md [hunk 1]
```diff
@@ -56,22 +56,13 @@ Feature  | Reports Synchronous (GET endpoint) | Reports Asynchronous (POST endpo
...
+> From the 1st of October 2020 Vonage is introducing two pricing options for all new Reports API customers.
 
-Charges are applied based on the quantity of requests and the volume of data retrieved as listed in the following table:
+Charges are applied based on the volume of data retrieved on the Pay As You Go as follows:
 
-Resource | Price
----- | ----
-Pay As You Go (PAYG) report generation | 0.0004€ per CDR
-Unlimited report generation | 450€ per month
+> 0.40€ per 1000 CDRs
```
Comment (mheap) [inline _documentation/en/reports/overview.md:63]: Is this charged per CDR, or in blocks of 1000? e.g. if I fetch 1 CDR is that 0.40 or 0.0004? If I fetch 1001 CDRs is that 0.4004 or 0.8?

**Comment by tbedford** [comment]
@adamczopp - some good questions from Michael to address please.

File: _documentation/en/reports/overview.md (hunk 1 above)
Comment (tbedford) [reply _documentation/en/reports/overview.md:63]: I was also wondering why we changed that. The examples have not changed, but I'm sure @adamczopp can clarify.

**Comment by tbedford** [comment]
@adamczopp - I also added a question on the JIRA ticket.

File: _documentation/en/reports/overview.md (hunk 1 above)
Comment (adamczopp) [reply _documentation/en/reports/overview.md:63]: Marketing asked to present it that way. It's still in the increments of 1 CDR (0.0004) but apparently that's how they want to present pricing for products where unit price is tiny. I thought we can clarify that through examples but maybe we can leave the line 0.0004€ per CDR and just add the new one (0.40€ per 1000 CDRs) below or in brackets?

**Comment by tbedford** [comment]
@adamczopp - I made some changes to clarify. I can't re-request you as a reviewer for some reason, but let me know what you think. Thanks.

**Review by adamczopp — APPROVED** [review]

**Review by mheap — APPROVED** [review]

**Pull request merged**